.github/workflows/unittests.yml

```diff
@@ -52,11 +57,20 @@ jobs:
 
   # Github Actions block ICMP. We can still use it for non root tests
   utscapy:
-    name: Non-sudo unit tests
-    runs-on: ubuntu-latest
+    name: ${{ matrix.os }} ${{ matrix.python }} ${{ matrix.mode }}
+    runs-on: ${{ matrix.os }}
     strategy:
       matrix:
+        os: [ubuntu-latest]
         python: [2.7, pypy2, pypy3, 3.5, 3.6, 3.7, 3.8, 3.9]
+        mode: [non_root]
+        include:
+          - os: macos-latest
+            python: 2.7
+            mode: both
```

Comment:
Don't you think that we can run root and non_root test separately?

Reply:
I don't think this is necessary. This is already how it was done, it would be a waste of resource to have to reinstall all dependencies